components/button/overview.md

```diff
@@ -44,95 +39,54 @@ To add a Telerik Button to your Blazor app, use the `<TelerikButton>` tag:
 }
 ````
 
->caption The result from the code snippet above
-
-![](images/basic-button.png)
-
->caption Component namespace and reference
-
-````CSHTML
-@using Telerik.Blazor.Components
-
-<TelerikButton @ref="theButton">Hello!</TelerikButton>
-
-@code{
-	Telerik.Blazor.Components.TelerikButton theButton;
-}
-````
-
-## Primary Button
+To see the result from the code snippet above, select the **PREVIEW** tab in the code snippet toolbar.
```

Comment:
Do we need to educate our readers for this? The UI should be user-friendly by itself.